js/binance.js

```diff
@@ -1690,7 +1694,12 @@ module.exports = class binance extends Exchange {
          * @returns {[object]} an array of objects representing market data
          */
         let promises = [];
-        const fetchMarkets = this.safeValue (this.options, 'fetchMarkets', [ 'spot', 'linear', 'inverse', 'option' ]);
+        const fetchMarkets = this.safeValue (this.options, 'fetchMarkets', [ 'spot', 'linear', 'inverse' ]);
+        const sandboxMode = this.safeValue (this.options, 'sandboxMode', false);
+        if (!sandboxMode) {
```

Comment:
@Dan-krm let me know if you find a better solution than this

Reply:
I looked through your changes and I can't think of a better alternative at the moment, I think there's a good chance binance will make their options testnet available again soon anyways considering they had it before